cli/internal/destinations/postgresql/README.md

```diff
@@ -0,0 +1,20 @@
+# CloudQuery PostgreSQL Destination Plugin
+
+This destination plugin let's you sync data from any CloudQuery source to any PostgreSQL compatible database.
+
+Known supported databases:
+
+- PostgreSQL > v10
+
+## PostgreSQL Spec
+
+This is the top level spec used by PostgreSQL Destination Plugin.
```

Comment:
When we write `top level`, what does it mean?

To me it means that the `yaml` file looks like this:

```
kind: destination
connection_string: <>
```

Reply:
we can maybe just omit the `top level` or even better just remove this line and go straight to the spec as it's a bit redundant. The user should be aware of our convention and that every plugin has a spec (it is mentioned in our main docs so no need to repeat that).

In any case it means the following:
```
kind: destination
spec:
  name: postgresql
  mode: overwrite
  spec:
     // goes here
     connection_string: stuff..

```

Similar to here - https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/ there is no intro wording that this is a spec and we already have an intro about what is the postgresql destination plugin